demo/app.js

```diff
@@ -551,17 +551,17 @@ const Demo = React.createClass({
               ref='drawer'
               title='This is the drawer component'
             >
-              <p style={{ padding: 20, fontFamily: 'Helvetica, Arial, sans-serif' }}>
+              <div style={{ padding: 20, fontFamily: 'Helvetica, Arial, sans-serif' }}>
```

Comment:
just so i'm understanding - this could have technically still been a `<p>` tag, but the ones below it would not have worked so you changed for consistency/DRYness